packages/ve-hemi-actions/actions/public/veHemi.ts

```diff
@@ -61,3 +62,45 @@ export const getOwnerOf = async function (
     functionName: 'ownerOf',
   })
 }
+
+export const getBalanceOfNFTAt = async function (
+  client: Client,
+  tokenId: bigint,
+  timestamp: bigint,
+): Promise<bigint> {
```

Comment:
I'm fairly sure we can drop this, as it should be inferred automatically by Typescript

```suggestion
) {
```